x/auth/keeper/keeper.go

```diff
@@ -60,6 +60,10 @@ var _ AccountKeeperI = &AccountKeeper{}
 
 // NewAccountKeeper returns a new AccountKeeperI that uses go-amino to
 // (binary) encode and decode concrete sdk.Accounts.
+// `maccPerms` is a map of module account to list of permissisons. It's used to construct
+// types.PermissionsForAddress and used in keeper.ValidatePermissions. Permissions is a string,
+// without any specific structure. It's not use internally by this module (x/auth) but may
+// be used by other modules using auth.Keeper to check against configured permissions.
 func NewAccountKeeper(
 	cdc codec.BinaryMarshaler, key sdk.StoreKey, paramstore paramtypes.Subspace, proto func() types.AccountI,
 	maccPerms map[string][]string,
```

Comment:
I was thinking to change the `maccPerms` type to `map[string]types.PermissionsForAddress`, but that would add redundancy when creating the parameters: `PermissionsForAddress` repeats the module name (the map key) in it's attributes.